spyder_kernels/console/shell.py

```diff
@@ -89,6 +89,10 @@ def enable_matplotlib(self, gui=None):
         if gui is None or gui.lower() == "auto":
             gui = automatic_backend()
 
+        # Before activating the backend, restore to file default those
+        # InlineBackend settings that may have been set explicitly.
+        self.kernel._restore_rc_file_defaults()
```

Comment:
```suggestion
        self.kernel.restore_rc_file_defaults()
```

To go with my suggestion above.